platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/properties/PropertiesLoaderTest.java

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.util.properties;
+
+import static org.mockito.Mockito.when;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PropertiesLoaderTest {
+
+  @Mock private Properties propertiesMock;
+
+  @ClassRule public static TemporaryFolder temporaryFolder = new TemporaryFolder();
+  private static final String PROPERTIES_FILENAME = "test.properties";
+  private static final String NON_EXISTENT_FILENAME = "lakjdslfkjadlkfn";
+  private static File propertiesFile;
+
+  private static Map<Object, Object> exampleMap;
+
+  private static final PropertiesLoader PROPERTIES_LOADER = PropertiesLoader.getInstance();
+
+  @BeforeClass
+  public static void setUpClass() throws Exception {
+    // initialize map and set
+    exampleMap = new HashMap<>();
+    exampleMap.put("a", "A");
+    exampleMap.put("b", "B");
+    exampleMap.put("c", "C");
+
+    // set up string to put in properties file
+    StringBuilder sb = new StringBuilder();
+    for (Map.Entry<Object, Object> entry : exampleMap.entrySet()) {
+      sb.append(entry.getKey() + "=" + entry.getValue() + "\n");
+    }
+
+    // initialize file
+    propertiesFile = temporaryFolder.newFile(PROPERTIES_FILENAME);
+    Files.write(propertiesFile.toPath(), sb.toString().getBytes());
+  }
+
+  @Test
+  public void testToMapWithProperties() throws Exception {
+    when(propertiesMock.entrySet()).thenReturn(exampleMap.entrySet());
+
+    Map<?, ?> entries = PROPERTIES_LOADER.toMap(propertiesMock);
+    Assert.assertEquals("Returns a map of the properties", exampleMap, entries);
```

Comment:
✏️  You can import `static org.junit.Assert.assertEquals` and directly call `assertEquals` in all your tests.